src/components/GameAbout/GameAbout.module.scss

```
@@ -1,32 +1,119 @@
+.container {
+  display: flex;
+  width: 75rem;
+  padding: var(--space-2xl) 7.5rem;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--space-lg);
+
+  @media (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 100%;
+    padding: var(--space-lg) var(--space-lg);
+  }
+}
+
 .iconOpen {
   transform: rotate(180deg);
 }
 
 .clampedText {
   display: -webkit-box;
+  -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
-  -webkit-line-clamp: 6;
   overflow: hidden;
 }
 
+.title {
+  color: var(--color-neutral-200);
+  margin: var(--space-md) 0;
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.titleLarge {
+  color: var(--color-neutral-200);
+  margin: var(--space-xs) 0;
+  font-weight: 600;
+}
+
+.titleSmall {
+  color: var(--color-neutral-200);
+  margin: var(--space-xs) 0;
+  font-weight: 600;
+}
+
 .description {
-  white-space: pre-line;
+  color: var(--color-neutral-400);
+  margin: 0;
 }
 
-.readMore {
+.showMore {
   display: flex;
-  gap: var(--space-xs);
-  align-items: center;
+  margin: var(--space-xs) 0;
   padding: 0;
+  justify-content: center;
+  align-items: center;
+  background: transparent;
+  border: none;
+  color: var(--color-primary-400);
+  cursor: pointer;
+  transition: background-color 0.2s ease;
 
-  svg {
-    fill: white;
-    width: 16px;
-    height: 16px;
+  &:hover {
+    color: var(--color-primary-300);
   }
 }
 
 .gameTitle {
-  padding-bottom: var(--space-md);
-  color: var(--color-neutral-200);
+  margin-bottom: 24px;
```

```suggestion
  margin-bottom: var(--space-2lg-fixed);
```
If you don't want it to scale off the title font size

If you do, remove the `-fixed`